components/alert/alert.css

```diff
@@ -0,0 +1,43 @@
+.comp-flex>.comp-display{
+    flex-direction:column;
+    align-items: flex-start;
+    
+}
+
+.comp-flex{
+    margin: 0rem auto;    
+}
+
+.alert{
+    font-size: var(--text-size);
+    border: 2px solid ;
+    width:100%;
+    border-radius:var(--corner);
+    padding: 0.4rem 1rem;
+    margin-bottom: 0.5rem;
+}
+
+i {
+    margin-right: 0.5rem;
+}
+
+.primary{
+    background-color: #e1adff;
```

Comment:
Use CSS color variables here.

Reply:
This is for specific element i.e. primary alert not for whole file.